lib/widgets/chat_list_welcome_tile.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:sloth/extensions/build_context.dart';
+import 'package:sloth/hooks/use_user_metadata.dart';
+import 'package:sloth/src/rust/api/welcomes.dart' show Welcome;
+import 'package:sloth/widgets/wn_animated_avatar.dart';
+
+class ChatListWelcomeTile extends HookWidget {
```

Comment:
🦥 : I'm struggling with the location of this widget... if it is just used in chat list, maybe it belongs in that file? cause widgets/ folder is currently used more like for shared content.... but at the same time, that screen file would easily grow a lot and I don't want that..